src/components/LoadingSpinner/LoadingSpinner.css

```diff
@@ -6,13 +6,22 @@
   }
 }
 
+.loading {
+  position: fixed;
+  display: grid;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  align-items: center;
+  justify-items: center;
```

Comment:
Try this instead:

```
.loading {
  display: flex;
  height: 100vh;
  align-items: center;
  justify-content: center;
}
```

Reply:
This was for the same overlap purpose, idk if its right